src/view/app/ModuleDetails/profilerModuleDetails.tsx

```diff
@@ -91,6 +102,9 @@ function getComparator(sortColumn: string) {
     case "totalTime":
     case "pcntOfSession":
     case "callerPcntOfSession":
+      return (a, b) => {
+        return a[sortColumn] - b[sortColumn];
```

Comment:
This is the same condition as for the case below, so this new statement is not needed